components/director/pkg/tenant/entity.go

```diff
@@ -0,0 +1,28 @@
+package tenant
+
+type Entity struct {
```

Comment:
Where is used this entity?

Reply:
It was intended to be a common entity between the director and the tenant fetcher. But I have forgotten to apply it on both components. Will do it right away.